lib/optimize.js

```diff
@@ -7,7 +7,10 @@
  *   - {Error} error
  *   - {Mix} value
  */
-exports.try = function (fn) {
+
+// !!! Typescript does not allow the use of reserved keywords as function names, and at present I have not found an effective solution, so I replace it with ' _ '.
+
+exports._try = function (fn) {
```

Comment:
> this is a breaking change? I don't think we should change this function name.

well, since this is a reserved keyword, TS does not have a clear compilation option to disable, I did not find an effective solution ...